src/main/java/com/dao/homework/dao/jdbc/ManufacturerJdbcDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package com.dao.homework.dao.jdbc;
+
+import com.dao.homework.dao.ManufacturerDao;
+import com.dao.homework.exceptions.DataProcessingException;
+import com.dao.homework.lib.Dao;
+import com.dao.homework.model.Manufacturer;
+import com.dao.homework.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
```

Comment:
we should remove Dao annotation from ManufacturerDaoImpl class

Reply:
be attentive, we should remove Dao from **ManufacturerDaoImpl** not from **ManufacturerJdbcDaoImpl**